src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.test.tsx

```diff
@@ -109,26 +120,33 @@ describe('<CheckboxGrid />', () => {
       }
     });
 
-    expect(wrapper.find(Checkbox).length).toBe(totalCheckedCheckboxes);
+    expect(container.querySelectorAll('.defaultCheckbox').length).toBe(
+      totalCheckedCheckboxes
+    );
   });
 
   it('hides the title when hideTitles is true', () => {
-    wrapper = mount(<CheckboxGrid {...defaultProps} hideLabel={true} />);
+    const { container } = renderCheckboxGrid({ hideLabel: true });
```

Comment:
line 128 talks about "title" and `hideTitles` 🤷‍♂️

Reply:
Updated the description